### PR TITLE
Update CsTeamsComplianceRecordingPolicy documentation due to rollback

### PR DESCRIPTION
**Pull request by abnan**

### CODE REVIEW

File: skype/skype-ps/skype/New-CsTeamsComplianceRecordingPolicy.md [hunk 1]
```diff
@@ -166,7 +166,7 @@ Accept wildcard characters: False
 ```
 
 ### -RecordReroutedCalls
-Setting this attribute to true enables compliance recording for calls that have been re-routed from a compliance recording-enabled user. Supported call scenarios include forward, transfer, delegation, call groups, and simultaneous ring.
+Setting this attribute to true enables compliance recording for calls that have been re-routed from a compliance recording-enabled user. Supported call scenarios include forward, transfer, delegation, call groups, and simultaneous ring. (Note: Only available in MicrosoftTeams PowerShell version 5.1.1-preview)
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsTeamsComplianceRecordingPolicy.md:169]:
```suggestion
Setting this attribute to true enables compliance recording for calls that have been re-routed from a compliance recording-enabled user. Supported call scenarios include forward, transfer, delegation, call groups, and simultaneous ring. (Only available in Microsoft Teams PowerShell version 5.1.1-preview.)

```

File: skype/skype-ps/skype/Set-CsTeamsComplianceRecordingPolicy.md [hunk 2]
```diff
@@ -225,7 +225,7 @@ Accept wildcard characters: False
 ```
 
 ### -RecordReroutedCalls
-Setting this attribute to true enables compliance recording for calls that have been re-routed from a compliance recording-enabled user. Supported call scenarios include forward, transfer, delegation, call groups, and simultaneous ring.
+Setting this attribute to true enables compliance recording for calls that have been re-routed from a compliance recording-enabled user. Supported call scenarios include forward, transfer, delegation, call groups, and simultaneous ring. (Note: Only available in MicrosoftTeams PowerShell version 5.1.1-preview)
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsTeamsComplianceRecordingPolicy.md:228]:
```suggestion
Setting this attribute to true enables compliance recording for calls that have been re-routed from a compliance recording-enabled user. Supported call scenarios include forward, transfer, delegation, call groups, and simultaneous ring. (Only available in Microsoft Teams PowerShell version 5.1.1-preview.)

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @abnan, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Review by dariomws — COMMENTED** [review]
Hi @abnan, my apologies, but I don't understand the purpose of this PR. Can you explain it?

For example, I cannot see this parameter in the GA or Preview version of the TPM. Maybe it should be removed from the articles.

Thanks!

**Comment by abnan** [comment]
@dariomws the parameter added should be available in version 5.1.1-preview. Due to a rollback by the Teams powershell team this parameter got removed in versions 5.2.0 and onward. They plan on re-releasing it, but meanwhile I am updating the documentation to capture which version to use for availing this parameter. The workaround (to use version 5.1.1-preview) for the time-being has been suggested by the Teams powershell team.

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**